Snippets/Core/Core_7/Handlers/DataAccess/ManualIdempotence.cs

```diff
@@ -0,0 +1,38 @@
+namespace Core7.Handlers.DataAccess
+{
+    using System;
+    using System.Threading.Tasks;
+    using NServiceBus;
+
+    public class ManualIdempotence : IHandleMessages<MyMessage>
+    {
+        #region BusinessData-ManualIdempotence
```

Comment:
This example is meant to convey what not to do correct?  I think this should be clarified so users don't try and do this.